docs/components/collapse.md

```diff
@@ -269,17 +270,29 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-.collapse-margin-bottom {
+@mixin margin-spacing {
   margin-bottom: var(--kui-space-50, $kui-space-50) !important;
 }
 
-.icon-margin-right {
+@mixin icon-spacing {
   margin-right: var(--kui-space-40, $kui-space-40) !important;
 }
 
-.trigger-content-flex {
+@mixin display-type {
```

Comment:
I think these mixins shoul live in `src/styles/_mixins.scss`. That way you can add an import of `@import '@/styles/mixins';` at the start of this style block and those mixins will be available in any other file where we might want to use them. Is that what you were thinking too @adamdehaven? If yes, then names of these mixins will need to be tweaked a little, but first let's make sure we meant the same thing.

Reply:
These mixins don't need to be mixins as it's not really helpful to map one-off style rules to a mixin.

Reply:
Talked to Adam, we don't need mixins for 1 line style properties. Moved the styles under the classes.